development/nautobot_config.py

```diff
@@ -92,6 +92,8 @@
 
 HIDE_RESTRICTED_UI = os.environ.get("HIDE_RESTRICTED_UI", False)
 
+CRYPTOGRAPHY_KEY = os.environ.get("NAUTOBOT_CRYPTOGRAPHY_KEY")
```

Comment:
Just asserting that without a fallback value this will default to `None`, and for development that is fine, right?

Reply:
If it falls back to None, startup will fail, which is why I added a setting for this variable to `dev.env` so that it shouldn't be unset for development. Is that ok, or is there somewhere else I need to set this as well?

Reply:
No, that's good.